saltproc/fuel_comp_025U

```diff
@@ -0,0 +1,7 @@
+mat fuel    -3.266364 burn 1 rgb 253 231 37 
```

Comment:
I feel this shouldn't be added to the toplevel directory. Perhaps there is  a more organized place to place it.